### DISCUSSION TITLE
Tuning advices

### DISCUSSION THREAD

**Discussion by itspers** [JushBJJ/Mr.-Ranedeer-AI-Tutor#59] [Q&A]
I tried to use it to learn portuguese language, but with example prompt `/plan I want to learn Japanese, but I have no experience with it.` it generates very basic behaviour, which basically spams you with information and tell to practice yourself after each lesson. Basic chat i started some days ago with prompt "please teach me portguese from the ground" basically do the same, but with less visual noise.

Maybe somebody have some advice of how to tune it, to be less like you bought book with 'practice yourself' after each chapter?

**Comment by JushBJJ** [comment]
In the prompt there are the lines:
```
            say **Let's start with an example:** <generate a random example problem>
            say **Here's how we can solve it:** <answer the example problem step by step>

            say Or **/test** to learn more **by doing**
```

Are you suggesting this should be optional to turn off?
